src/core/search/search.cc

```diff
@@ -298,27 +311,46 @@ struct BasicSearch {
     return IndexResult{};
   }
 
+  // SORTBY [DESC] clause
```

Comment:
It looks like Redis' API is `SORTBY clause [DESC]` (i.e. `DESC` should be last): https://redis.io/docs/interact/search-and-query/advanced-concepts/sorting/#specifying-sortby

Reply:
clause is just an part of the comment that means its not part of the query but a separate clause